test/k8sT/CustomCalls.go

```diff
@@ -115,6 +110,10 @@ var _ = Describe("K8sCustomCalls", func() {
 			ExpectAllPodsTerminated(kubectl)
 		})
 
+		AfterAll(func() {
+			deploymentManager.DeleteAll()
```

Comment:
It seems this is not exactly the same as:
```
 deploymentManager.DeleteCilium()
 deploymentManager.DeleteAll()
```
and doesn't remove the Cilium ConfigMap.

As a result, the few tests following K8sCustomCalls in the net-next job are failing to install Cilium with:
```
Error: rendered manifests contain a resource that already exists. Unable to continue with install: existing resource conflict: kind: ConfigMap, namespace: kube-system, name: cilium-config
```

Reply:
I see, thanks for looking into it! I added the call to `DeleteCilium()` back, hopefully this time it should work better.

Reply:
Note the order is now inverted with this PR. Hopefully, it shouldn't matter.